src/pages/NervosDao/DaoOverview/index.tsx

```diff
@@ -209,19 +209,19 @@ const useOption = (nervosDao: State.NervosDao, colors: string[], isMobile: boole
       title: t('nervos_dao.burnt'),
     },
   ]
-  const selectedData: any = {
-    first: true,
-  }
-  selectedData[names[0]] = true
-  selectedData[names[1]] = true
-  selectedData[names[2]] = true
 
   return {
     color: colors,
     tooltip: {
       trigger: 'item',
-      formatter: (value: any) =>
-        `${value.data.title}: ${localeNumberString(value.data.value)} ${t('common.ckb_unit')} (${value.data.name})`,
+      formatter: value => {
+        if (Array.isArray(value)) {
+          throw new Error('value should not be array')
+        }
```

Comment:
Why not validate if `value` has a valid `data` field

Reply:
The following usage is like:
<img width="953" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/3870972/7ab3ca5e-1c91-4e72-a0b3-28a88cd09cbe">

treating `value` as an object, by doing this we narrow down the type of real `value`

Reply:
The validation is incomplete. It guarantees that `value` is not an array, but the usage of `value` is retrieving `value.data`. The validation is expected to make sure `value` has a valid field `data`, and `data` consists of `title`, `value`, and `name`

Reply:
The signature of `formatter` is `(value: Format | Format[]) => string` 

By the assertion, narrows down it's type from `Format | Format[]` to `Format`.

And `Format` has this `data` attribute